src/preact/component/renderer.js

```diff
@@ -27,9 +33,17 @@ export function useRenderer(renderer, data) {
         canceled = true;
       };
     } else {
-      setValue(/** @type {?RendererFunctionResponseType} */ (rendered));
+      setValue(/** @type {VNode} */ (rendered));
     }
   }, [renderer, data]);
 
   return value;
 }
+
+/**
+ * @param {*} obj
+ * @return {obj is Promise<*>}
+ */
+function isPromise(obj) {
```

Comment:
This might belong in `#core/types`, but I don't have strong feelings

Reply:
Moved